### PR TITLE
add notes and limitations to README

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,5 +25,58 @@ DATABASES = {
...
+4. Migrations have some limitations. CockroachDB doesn't support:
+
+   1. [changing column type](https://github.com/cockroachdb/cockroach/issues/9851)
+   2. dropping or changing a table's primary key
+   3. ...
+
+5. Incorrect queries:
+   1. [extract() function doesn't respect active time zone](https://github.com/cockroachdb/django-cockroachdb/issues/47)
+   2. [Incorrect interval math across daylight saving time](https://github.com/cockroachdb/django-cockroachdb/issues/54)
+   3. [`DATE_TRUNC('week', <value>)` truncates to midnight Sunday rather than
```
Comment (timgraham) [inline README.md]: Fixed on master

File: README.md [hunk 2]
```diff
@@ -17,5 +25,58 @@ DATABASES = {
...
+   aren't supported.
+
+4. Migrations have some limitations. CockroachDB doesn't support:
+
+   1. [changing column type](https://github.com/cockroachdb/cockroach/issues/9851)
+   2. dropping or changing a table's primary key
+   3. ...
+
+5. Incorrect queries:
+   1. [extract() function doesn't respect active time zone](https://github.com/cockroachdb/django-cockroachdb/issues/47)
```
Comment (timgraham) [inline README.md]: Fixed on master

File: README.md [hunk 3]
```diff
@@ -17,5 +23,73 @@ DATABASES = {
...
+   `TextField` so `CharField` is introspected by `inspectdb` as `TextField`.
+
+2. `IntegerField` uses the same storage as `BigIntegerField` so `IntegerField`
+   is introspected by `inspectdb` as `BigIntegerField`.
+
+3. `AutoField` and `BigAutoField` are both stored as
+   [integer](https://www.cockroachlabs.com/docs/stable/int.html) (64-bit) with
+   [`DEFAULT unique_rowid()`](https://www.cockroachlabs.com/docs/stable/functions-and-operators.html#id-generation-functions).
+
+## Known issues and limitations (as of cockroachdb 19.2.1)
```
Comment (rafiss) [inline README.md]: Unfortunately, the fixes for these issues aren't in 19.2.1,  but we plan to get them into 19.2.2

- https://github.com/cockroachdb/django-cockroachdb/issues/13
- https://github.com/cockroachdb/django-cockroachdb/issues/97
- https://github.com/cockroachdb/django-cockroachdb/issues/53
- https://github.com/cockroachdb/django-cockroachdb/issues/47
- https://github.com/cockroachdb/django-cockroachdb/issues/37
- https://github.com/cockroachdb/django-cockroachdb/issues/32

I think that should be everything, but the 19.2.1 release notes are here: https://www.cockroachlabs.com/docs/releases/v19.2.1.html

Comment (timgraham) [reply README.md]: Right, I added all those here this morning. Although I see I've added 47 twice.

Comment (rafiss) [reply README.md]: Oops I missed those. LGTM after fixing the duplicate!

**Pull request merged**